cmd/release-controller/sync.go

```diff
@@ -482,6 +482,12 @@ func (c *Controller) syncReady(release *Release) error {
 		if names, ok := status.Failures(); ok {
 			if allOptional(release.Config.Verify, names...) {
 				glog.V(4).Infof("Release %s had only optional job failures: %v", releaseTag.Name, strings.Join(names, ", "))
+			} else if retryNames, ok := status.retriesRemaining(release.Config.Verify); ok {
```

Comment:
you should restructure this block to be more readable.

```
if retryNames, ok := ...; ok {
  ...
  continue
}
if !allOptional(...) {
  ...
  continue
}
glog.V(4).Infof("Release %s had only optional job failures", ...)
```

That makes this much more readable.